src/styles/components/_form.scss

```diff
@@ -36,34 +38,72 @@
   &__input-label {
     display: block;
     margin-bottom: rem(12px);
+
+    font-size: rem(16px);
+    font-weight: 600;
+    line-height: 1.2;
+
+    @include respond-above('lg') {
+      font-size: rem(20px);
+    }
   }
 
   &__input-group {
     width: 100%;
   }
 
-  &__actionable,
-  &__recaptcha {
+  &__input-password {
+    position: relative;
     display: flex;
-    justify-content: center;
+    flex-direction: column;
+    box-sizing: border-box;
   }
 
   &__input {
-    width: 100%;
+    display: block;
     box-sizing: border-box;
-    padding: rem(4px 12px);
+    width: 100%;
+    padding: 0 0 rem(8px);
+    border-color: $havelock-blue;
+    border-width: 0 0 rem(2px);
+    outline: 0;
+
+    font-size: rem(16px);
+
+    &--invalid {
+      border-color: $coral-red;
+    }
   }
 
   &__button {
-    color: $light;
-    text-decoration: none;
-    margin: 0;
     padding: rem(5px 10px);
     border-radius: rem(3px);
+    margin: 0;
+
     background-color: $shuttle-gray;
+    color: $light;
+    text-decoration: none;
 
     &:hover {
       background-color: darken($silver, 8%);
     }
   }
+
+  &__help {
+    display: none;
+    
+    &--visible {
+      display: block;
+      margin-top: rem(6px);
+      color: $coral-red;
+      font-size: rem(14px);
+    }
+  }
```

Comment:
We could use `React` to handle the [conditional rendering](https://reactjs.org/docs/conditional-rendering.html) logic